src/components/vite.config.ts

```diff
@@ -1,5 +1,6 @@
 import { dirname, join } from 'path';
 
+// @ts-expect-error FIXME Typescript error
```

Comment:
What happens here?

Reply:
There is this error:

![image](https://github.com/lyne-design-system/lyne-components/assets/101575400/aa484e79-8632-4b78-b35f-9d724d6ca75b)

trying the `npm i blabla` solution, it fails with a "conflicting peer dependency" error related to `@open-wc/lit-helpers`

Reply:
Does @kyubisation have an idea here?

Reply:
It worked 🚀